Assets/_BForBoss/Scripts/Debug/DebugView/PlayerPrefResetter.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class PlayerPrefResetter : DebugView
+    {
+        private Vector2 _scrollPosition = Vector2.zero;
+
+        private IList<string> _keys;
+
+        public PlayerPrefResetter(Rect masterRect) : base(masterRect)
+        {
+            _keys = GetConstStringValuesFromStruct<UploadPlayerScoreDataSource.PlayerPrefKey>();
+            // if we add player pref fields anywhere else make sure to append them to the array here
```

Comment:
There's actually more than a few places with PlayerPrefKeys

Reply:
https://perigongames.atlassian.net/browse/BFB-75

I created a ticket to add them in, for the future